<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>distanceSensorUltrasonicLCD</class>
 <widget class="QMainWindow" name="distanceSensorUltrasonicLCD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1438</width>
    <height>978</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>distanceSensorUltrasonicLCD</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>121</width>
      <height>591</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QwtSlider" name="QwtSliderDistance">
       <property name="upperBound">
        <double>400.000000000000000</double>
       </property>
       <property name="value">
        <double>10.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QwtPlot" name="QwtPlotDistance">
    <property name="geometry">
     <rect>
      <x>327</x>
      <y>164</y>
      <width>961</width>
      <height>631</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>961</width>
      <height>631</height>
     </size>
    </property>
    <property name="autoReplot">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QwtTextLabel" name="QwtTextLabelDistance">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>440</y>
      <width>147</width>
      <height>77</height>
     </rect>
    </property>
    <property name="plainText">
     <string>Value</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>460</y>
      <width>149</width>
      <height>79</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1438</width>
     <height>39</height>
    </rect>
   </property>
   <widget class="QMenu" name="menudistanceSensorUltrasonicLCD">
    <property name="title">
     <string>dista&amp;nceSensorUltrasonicLCD</string>
    </property>
   </widget>
   <addaction name="menudistanceSensorUltrasonicLCD"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
  <customwidget>
   <class>QwtSlider</class>
   <extends>QWidget</extends>
   <header>qwt_slider.h</header>
  </customwidget>
  <customwidget>
   <class>QwtTextLabel</class>
   <extends>QFrame</extends>
   <header>qwt_text_label.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
